src/event-handlers/kleros-liquid-emails.js

```diff
@@ -45,15 +45,19 @@ const handlers = {
             account: event.returnValues._address,
             message: `Congratulations! You have been drawn as a juror on case #${
               event.returnValues._disputeID
-            }. Voting starts ${timeAgo.format(
-              (Number(dispute.lastPeriodChange) +
-                Number(
-                  (await klerosLiquid.methods
-                    .getSubcourt(dispute.subcourtID)
-                    .call()).timesPerPeriod[0]
-                )) *
-                1000
-            )}`,
+            }. Voting starts ${
+              event.returnValues._appeal === '0'
+                ? timeAgo.format(
+                    (Number(dispute.lastPeriodChange) +
+                      Number(
+                        (await klerosLiquid.methods
+                          .getSubcourt(dispute.subcourtID)
+                          .call()).timesPerPeriod[0]
+                      )) *
+                      1000
+                  )
+                : 'as soon as all other jurors are drawn'
```

Comment:
"Voting starts once all jurors have been selected" is more laymen's

Reply:
"Select" kind of implies someone deciding.

We use the term "Draw" everywhere.

Reply:
I don't see Select that way but it is fine if we use Draw other places as well.

Reply:
Yeah, consistency is more important here.